From c8b2f218c7a9a5b9bb416b21dfe561653cd7d8c1 Mon Sep 17 00:00:00 2001
From: Petr Uzel <petr.uzel@suse.cz>
Date: Fri, 28 May 2010 16:42:20 +0200
Subject: [PATCH] parted: do_align_check returns always 1 in interactive mode

* parted/parted.c (do_align_check): Always return 1 in interactive mode.
---
 parted/parted.c | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/parted/parted.c b/parted/parted.c
index 7723d18..265282a 100644
--- a/parted/parted.c
+++ b/parted/parted.c
@@ -2094,8 +2094,12 @@ do_align_check (PedDevice **dev)
 
   ped_disk_destroy (disk);
 
-  /* FIXME: perhaps we should always return 1 when in interactive mode???  */
-  return aligned ? 1 : 0;
+  if (opt_script_mode)
+    return aligned ? 1 : 0;
+
+  /* Always return 1 in interactive mode, to be consistent
+     with the other modes.  */
+  return 1;
 
 error_destroy_disk:
   ped_disk_destroy (disk);
-- 
1.9.0

